kroxylicious/pom.xml

```diff
@@ -319,5 +321,8 @@
                 </dependency>
             </dependencies>
         </profile>
+        <profile>
```

Comment:
this should be removed, it was part of a different approach I tried and is dead code